### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeevan-jp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # MyL3-External-Data
 GSoC 2018 project
+
+Idea is to develop basic Infusion wrapper for My Life Long Learning Lab(MyL3). This wrapper will contain following data about learners imported from various external sources.
```
Comment (sepidehshahi) [inline README.md]: This project aims to develop ....

Comment (simonbates) [inline README.md]: I would maybe say something like:

"This project aims to develop external data sources for the My Life Long Learning Lab (MyL3) project.

Data sources that will be researched and prototyped include:"

I don't think I would describe the project as a "wrapper".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+
+* Import Weather data
+* Import Map/Location data
+* Import Google Fit data
+* Import FitBit data
+* Import Facebook Status
+
+More info about MyL3: [click here](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment)
```
Comment (simonbates) [inline README.md]: I would prefer a more descriptive link text than "click here". Maybe something like:

More information about the project may be found on the [MyL3 page of the Fluid wiki](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment).

File: README.md (hunk 1 above)
Comment (jeevan-jp) [reply README.md]: Thanks,
Updated in second commit.

Comment (jeevan-jp) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (jeevan-jp) [reply README.md]: Updated

**Review by sepidehshahi — APPROVED** [review]

**Pull request merged**